yt/geometry/particle_geometry_handler.py

```diff
@@ -174,8 +175,14 @@ def _initialize_index(self):
         # Load Morton index from file if provided
         def _current_fname():
             if getattr(ds, "index_filename", None) is None:
-                fname = ds.parameter_filename + ".index{}_{}.ewah".format(
-                    self.regions.index_order1, self.regions.index_order2
+                range = f"[{self.regions.index_order2}-{self.regions.index_order2+2}]"
```

Comment:
I'd rather avoid shadowing builtins (suggesting `irange` for "index range" here, but basically any name works)
```suggestion
                irange = f"[{self.regions.index_order2}-{self.regions.index_order2+2}]"
```